ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/TableOptimizeItem.java

```diff
@@ -445,7 +464,8 @@ private FilesStatistics collectOptimizeFileInfo(Collection<BaseOptimizeTask> tas
     return builder.build();
   }
 
-  private void tryUpdateOptimizeInfo(TableOptimizeInfo.OptimizeStatus optimizeStatus,
+  private void tryUpdateOptimizeInfo(
+      TableOptimizeRuntime.OptimizeStatus optimizeStatus,
                                      Collection<BaseOptimizeTask> optimizeTasks,
                                      Set<OptimizeType> types) {
```

Comment:
There is no need to use a `Set` of types

Reply:
Fixed it.